docs/modules/servers/pages/local/conf/index.adoc

```diff
@@ -1,23 +1,99 @@
 = James Local Mail Server Configuration
 :navtitle: Configuration
 
+== Configuration by feature
+
+ * Domain names &mdash; provide the default domain names that ????
+ * Database (james-database.properties)
+ * LMTP (lmtpserver.xml)
+ * POP3 (pop3server.xml)
+ * DNS (dnsservice.xml)
+ * Loggin (logback.xml)
+ * James CLI (jmx.properties)
+ * Email rewriting (recipientrewritetable.xml)
+ * Guice extensions (extensions.properties)
+ * JWT (jwt_publickey)
+ * Mailets (mailetcontainer.xml)
+ * SMTP (smtpserver.xml)
+ * Health checks (healthcheck.properties)
+ * Mail storage (mailrepositorystore.xml)
+ * User storage (usersrepository.xml)
+ * IMAP (imapserver.xml)
+ * Mailbox event listeners (listeners.xml)
+ * Sieve filters (managesieveserver.xml)
+ * Web admin (webadmin.properties)
+
+== Configuration file index
+
  * Domain names (domainlist.xml)
  * Database (james-database.properties)
  * LMTP (lmtpserver.xml)
  * POP3 (pop3server.xml)
  * DNS (dnsservice.xml)
  * Loggin (logback.xml)
- * ??? (jmx.properties)
- * ??? (recipientrewritetable.xml)
- * ??? (extensions.properties)
+ * James CLI (jmx.properties)
+ * Email rewriting (recipientrewritetable.xml)
+ * Guice extensions (extensions.properties)
  * JWT (jwt_publickey)
  * Mailets (mailetcontainer.xml)
  * SMTP (smtpserver.xml)
  * Health checks (healthcheck.properties)
  * Mail storage (mailrepositorystore.xml)
  * User storage (usersrepository.xml)
  * IMAP (imapserver.xml)
- * ??? (listeners.xml)
+ * Mailbox event listeners (listeners.xml)
  * Sieve filters (managesieveserver.xml)
  * Web admin (webadmin.properties)
+
+
+== Notes
+Jmx protocol is used by James cli to interactwith the server. Jmx is known to be insecure, and we have as a project to rewrite the cli for james servers in order to rely on webadmin.
+
+
+
+Scenarios for address rewriting
```

Comment:
https://github.com/apache/james-project/blob/master/src/site/xdoc/server/config-recipientrewritetable.xml (not yet deployed on current website :-( tries to provide wording about Recipient Rewriting and use cases.

Reply:
Maybe `address rewriting` needs to be moved in the concept part?

Reply:
Yes, definitely. Anything that requires knowledge of any concept should be there.

Reply:
Added as a note